src/ontology/uberon-edit.obo

```diff
@@ -73542,7 +73536,6 @@ name: right lung bronchiole
 def: "A bronchiole that is part of a right lung [Automatically generated definition]." [OBOL:automatic]
 synonym: "bronchiole of right lung" EXACT [OBOL:automatic]
 synonym: "lobular bronchiole of right lung" EXACT [OBOL:automatic]
```

Comment:
Consider removing this synonym as well.

Reply:
Thank you for spotting this synonym. I have also removed a "mucosa of bronchiole" synonym.